lib/core/res.css

```diff
@@ -1204,7 +1195,7 @@ img {
 	max-width: 23px;
 	max-height: 23px;
 	display: inline-block;
-	background-image: url('https://s3.amazonaws.com/b.thumbs.redditmedia.com/tBwwK20XXxtpgudWx1L7bDXla-iotv-JA0jgA0Y-FVs.png');
+	background-image: url('https://i.imgur.com/oYDs89u.png');
```

Comment:
I reverted 3a8da02 on account of we maxed out the /r/enhancement stylesheet images and I'm not sure how to resolve it yet.